RELEASE_NOTES.rst

```diff
@@ -1,3 +1,132 @@
+coala-bears 0.9.0
+=================
+
+This release, we had 40 different contributors, pushing almost 200 commits.
+
+To get a quick overview over the current state of all bears, check out
+https://coala.io/languages.
+
+Here are the important changes and additions, of course coala was upgraded to
+0.9 so you can enjoy all it's new features and bugfixes:
+
+**New Bears:**
+
+Generic:
+
+- `QuotesBear` - Checks and corrects your quoting style for any language!
+
+Web:
+
+- `PHPCodeSnifferBear` - Ensures that your PHP, JavaScript or CSS code remains
+  clean and consistent.
+- `Jinja2Bear` - Detects and fixes formatting issues in Jinja2 templates.
```

Comment:
I don't know if this matters at all, but for what it's worth: Jinja2 is not relly limited to Web-Applications. As a templating language, it can be used for any text based file. I use it to generate .tex files for example. 

I would probably list it under "Others" or have a "Templating" section.